src/dialects/ibmi/query-interface.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const { Transaction } = require('../../transaction');
+const { QueryInterface } = require('../abstract/query-interface');
+
+/**
+ Returns an object that enables the `ibmi` dialect to call underlying odbc
+ transaction functions through the connection methods instead
+
+ @class QueryInterface
+ @static
+ @private
+ */
+
+class IBMiQueryInterface extends QueryInterface {
+
+  startTransaction(transaction, options) {
+    if (!transaction || !(transaction instanceof Transaction)) {
```

Comment:
could be simplified to just

```suggestion
    if (!(transaction instanceof Transaction)) {
```